### PR TITLE
Fixed wrong results while doing division

### LINKED ISSUE #2441: Wrong calculation of MTR
In the example below, the calculated MTR value is -53.41%, whereas it should be (633-303)/633=52%:
Screen Shot 2019-09-15 at 10 19 51 PM

Related to [Forum comment](http://forum.spinalcordmri.org/t/calculate-mtr-divide-by-zero-error/170/8)

### PR DESCRIPTION
**Pull request by jcohenadad**
The division as currently implemented is unstable and sometimes give wrong results. See e.g. #2441. This PR addresses the issue by replacing the division "/" by `np.true_divide`. More specifically:

DONE:
- Implemented the fix in `sct_compute_mtr`
- Implement the same fix in `sct_compute_mtsat` (was already implemented)
- Refactor all qMRI techniques (excluding diffusion) into the folder qmri, and put `compute_mtsat()` and `compute_mtr()` inside the same file `qmri/mt.py`
- Added unit tests across different MRI data types

Fixes #2441.

### CODE REVIEW

File: spinalcordtoolbox/qmri/mt.py [hunk 1]
```diff
@@ -34,9 +46,11 @@ def compute_mtsat(nii_mt, nii_pd, nii_t1,
     :return:
     """
     # params
-    nii_t1map = None  # it would be possible in the future to input T1 map from elsewhere (e.g. MP2RAGE). Note: this
-                      # T1map needs to be in s unit.
-    b1correctionfactor = 0.4  # empirically defined in https://www.frontiersin.org/articles/10.3389/fnins.2013.00095/full#h3
+    nii_t1map = \
```
Comment (gmotzespina) [inline spinalcordtoolbox/qmri/mt.py]: Why is this change needed?

File: scripts/sct_compute_mtr.py [hunk 2]
```diff
@@ -105,18 +57,26 @@ def get_parser():
...
+    # Check input parameters
+    parser = get_parser()
+    args = parser.parse_args(args=None if sys.argv[1:] else ['--help'])
+    fname_mtr = args.o
+    verbose = args.v
+    # Update log level
+    sct.init_sct(log_level=verbose, update=True)
+
+    # compute MTR
+    sct.printv('\nCompute MTR...', verbose)
```
Comment (gmotzespina) [inline scripts/sct_compute_mtr.py]: Should we leave this with double quotes as stated in the [proposed convention](https://github.com/neuropoly/spinalcordtoolbox/issues/2439#issuecomment-531304020) or should we wait until we fully agree on it?

File: scripts/sct_compute_mtsat.py [hunk 3]
```diff
@@ -126,9 +127,32 @@ def main():
     verbose = args.v
     sct.init_sct(log_level=verbose, update=True)  # Update log level
 
-    fname_mtsat, fname_t1map = mtsat.compute_mtsat_from_file(
-        args.mt, args.pd, args.t1, args.trmt, args.trpd, args.trt1, args.famt, args.fapd, args.fat1,
-        fname_b1map=args.b1map, fname_mtsat=args.omtsat, fname_t1map=args.ot1map, verbose=verbose)
+    sct.printv('Load data...', verbose)
```
Comment (gmotzespina) [inline scripts/sct_compute_mtsat.py]: Same question about the convention

File: scripts/sct_compute_mtsat.py [hunk 4]
```diff
@@ -126,9 +127,32 @@ def main():
...
+    else:
+        nii_b1map = Image(args.b1map)
+
+    # compute MTsat
+    nii_mtsat, nii_t1map = compute_mtsat(nii_mt, nii_pd, nii_t1, args.trmt, args.trpd, args.trt1, args.famt, args.fapd,
+                                         args.fat1, nii_b1map=nii_b1map)
+
+    # Output MTsat and T1 maps
+    # by default, output in the same directory as the input images
+    sct.printv('Generate output files...', verbose)
```
Comment (gmotzespina) [inline scripts/sct_compute_mtsat.py]: Same question about the convention

File: scripts/sct_compute_mtr.py (hunk 2 above)
Comment (jcohenadad) [reply scripts/sct_compute_mtr.py]: let's wait, and also try to minimize the number of cosmetic changes unrelated to this PR (although i did some...)

File: spinalcordtoolbox/qmri/mt.py (hunk 1 above)
Comment (jcohenadad) [reply spinalcordtoolbox/qmri/mt.py]: more inline with PEP8, right?

**Comment by jcohenadad** [comment]
sanity checking, `batch_processing.sh` results are ok:
~~~
t2/CSA:          73.98379456979737
mt/MTR(WM):      54.31259385601459
t2s/CSA_GM:      12.530353265289374
t2s/CSA_WM:      65.07686820036822
dmri/FA(CST_r):  0.7940208910003284
dmri/FA(CST_l):  0.7704311978658454
~~~
